### PR TITLE
Homework 03/ Ширшова Настя

### PR DESCRIPTION
**Pull request by captainger**

### CODE REVIEW

**Comment by Kinetikm** [comment]
Недельник 5/5 В пятнице можно было бы покрасивее сделать через groupby().value_counts(normalized=True) ПЛюс в groupby можно передавать словарь функций агрегации, в криминале норм зашло бы для популяции максимум 1 шагом. Фласк норм 5/5. Кнн 5/5 при train-test лучше стратифицировать через stratify. Отбор фичей можно делать автоматически при более сильных алгоритмах, например = борута, RFC (recursive feature что-то там в sklearn.feature_selection)
